src/main/java/view/SignupView.java

```diff
@@ -88,40 +155,89 @@ public void actionPerformed(ActionEvent evt) {
                                     currentState.getRepeatPassword(),
                                     currentState.getEmail(),
                                     currentState.getBirthdate(),
-                                    currentState.getFullName(),
-                                    null,
```

Comment:
Instead of null, ensure currentState properly initializes moderators and posts (e.g. as empty lists) to prevent potential NullPointerException